### PR TITLE
Docs Q2

### PR DESCRIPTION
**Pull request by lewish**
Previous PR + new content structure.
Redirects are in place for old URLs and I've gone through and checked all links with a link checker

### CODE REVIEW

**Comment by lewish** [comment]
Sitemap - this is deliberate, although agree it's a lot, but it's so ctrl+f works.
Cards not sidebyside - can't for the love of me work out how to change this. something to do with the way columns/masonry works wher there are only 2 elements.
Old logo, yeayea, Guillaume is working on it :D
Weird icons - fixed.
Grey omnibar - fixed.

File: docs/global.css [hunk 1]
```diff
@@ -37,3 +37,10 @@
 :global pre {
   font-size: 0.95em;
 }
+
+figcaption {
```
Comment (GJMcGowan) [inline docs/global.css:41]: This won't do anything as it's targeting an html element, I believe you want the class?

Comment (lewish) [reply docs/global.css:41]: No that is actually the intention! This is a real tag apparently, and Guillaume used it in the actual markdown files.

Comment (lewish) [reply docs/global.css:41]: (I removed it before, that's why it messed up the spacing)

Comment (GJMcGowan) [reply docs/global.css:41]: My god, you're right.

**Pull request merged**